### DISCUSSION TITLE
Project board

### DISCUSSION THREAD

**Discussion by wormstein** [EdNekebno/DGTCentaurMods#22] [Ideas]
Hi,
I think we should get more agile at this project by implementing a project board so we can keep track who's working on what.
This way we make sure we don't spend the time on something that someone else is already on it. I, for example, am thinking to implement a python envitonment.

I know we don't have too much time to invest into this but at least we make it worth.

I'd say to make a board with basic 3 columns: to-do, in progress and complete. Add us as collaborators. Here is how to do it: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-user-account-settings/managing-access-to-your-user-accounts-project-boards

What do y'all think?

**Comment by EdNekebno** [comment]
Sounds good to me....maybe I could use some focus :) My only concern is that it might go a bit slowly. However, it fits with the idea of a roadmap too. I'll see if my github cluelessness allows me to follow those instructions :)

**Comment by lucamonfredo** [comment]
I'm all for it, it should make our efforts clear and focused 👍

**Comment by EdNekebno** [comment]
It seemed to let me make one in the context of this repository, but then it doesn't give any collaboration options: https://github.com/EdNekebno/DGTCentaur/projects/1 . So then I thought I'd just make you both collaborators for the repository.
